python/pyspark/ml/tests.py

```diff
@@ -807,6 +807,18 @@ def test_logistic_regression(self):
         except OSError:
             pass
 
+    def logistic_regression_check_thresholds(self):
+        self.assertIsInstance(
+            LogisticRegression(threshold=0.5, thresholds=[0.5, 0.5]),
+            LogisticRegressionModel
```

Comment:
It shouldn't, but it's weird that Jenkins always passed the test. Anyway, let's fix it. Thanks for catching this.